webui/react/src/hooks/useMetricNames.ts

```diff
@@ -9,21 +9,21 @@ import { MetricName, MetricType } from 'types';
 export interface UseMetricNamesInterface {
   errorHandler: () => void;
   experimentId: number;
-  metricNames: MetricName[];
-  setMetricNames: (metrics: MetricName[]) => void;
 }
 
-const useMetricNames = (args: UseMetricNamesInterface): void => {
+const useMetricNames = (experimentId: number, errorHandler: () => void): MetricName[] => {
+  const [metrics, setMetrics] = useState<MetricName[]>([]);
```

Comment:
we can bring the `metricNames` state into the hook itself in order to simplify the interface.

Reply:
🙌  this works a lot better